upstream/CVE-2023-2727.json

```
@@ -0,0 +1,113 @@
+{
+  "id": "CVE-2023-2727",
+  "modified": "2023-06-13T14:42:06Z",
+  "published": "2023-06-13T14:42:06Z",
+  "summary": "Bypassing policies imposed by the ImagePolicyWebhook and bypassing mountable secrets policy imposed by the ServiceAccount admission plugin",
+  "details": "Users may be able to launch containers using images that are restricted by ImagePolicyWebhook when using ephemeral containers. Kubernetes clusters are only affected if the ImagePolicyWebhook admission plugin is used together with ephemeral containers.\n\n",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/apiserver"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.1/AV:N/AC:L/PR:H/UI:N/S:U/C:H/I:H/A:N"
+        }
+      ],
+      "ranges": [
+        {
+          "events": [
+            {
+              "introduced": "1.24.0"
+            },
+            {
+              "last_affected": "1.24.14"
```

It has a fixed version.
https://github.com/kubernetes/kubernetes/issues/118640

```suggestion
              "fixed": "1.24.15"
```